src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.datajpa.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
```

Comment:
data with entities is bad practice
see here why
https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/